winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
```

Comment:
Hmm that is odd, that javascript is generated by the `browserify` step which is a requirement of the `build` step which is part of the `default` step, so just running `gulp` *should* be doing the right thing.